Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarQubeWebServerTest.cs

```diff
@@ -70,22 +70,25 @@ public SonarQubeWebServerTest()
         [DataRow("8.8.0.1121")]
         [DataRow("9.0.0.1121")]
         [DataRow("10.15.0.1121")]
-        public void WarnIfDeprecated_ShouldNotWarn(string sqVersion)
+        public void WhenVersionIsDeprecated_ShouldNotWarn(string sqVersion)
```

Comment:
Do we still need this test? From what I can see we no longer have logger invocations in the constructors.

Reply:
Indeed, the test is now testing that the constructor is not emitting warnings, which is not relevant.
I have removed it.